### ISSUE TITLE
Have a problem with adding in project

### ISSUE THREAD

**Issue by dEROZA** [php-telegram-bot/core#1249]
`longman/telegram-bot 0.74.0 requires psr/log ^1.1 -> found psr/log[1.1.0, ..., 1.1.4]`
`but the package is fixed to 2.0.0 (lock file version)+`

composer req for symfony project, can't install tgbot

**Comment by pavanyogi** [comment]
You can rename composer.lock to composer.lock.bkp and try to install with the following command:
```sh
composer require -W  longman/telegram-bot

-W, --update-with-all-dependencies                   Allows all inherited dependencies to be updated, including those that are root requirements.
```

**Comment by TiiFuchs** [comment]
Looks like it could be fixed by #1282

**Comment by noplanman** [comment]
@dEROZA Version 0.76.0 was just released, it addresses this issue too 👍
